### DISCUSSION TITLE
제페토 월드에서 헤어스타일 변경

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#218] [5. Q&A]
안녕하세요.
유니티로 제작한 제페토 월드에서 특정 이벤트 발생 시 아바타의 헤어스타일 변경이 가능한가요?
예) 걸어가다가 함정에 걸리면 대머리로 변경 또는 특정 설정된 헤어스타일로 변경
가이드 부탁드립니다.
감사합니다.

**Comment by moondory77-naverz** [comment]
안녕하세요.
현재 제페토 월드개발에서는, 로드한 캐릭터에 대해 부위별 커스터마이징 옵션은 따로 제공하고 있지 않습니다.
감사합니다.
